src/status_im/ui/screens/profile/edit/views.cljs

```diff
@@ -54,24 +58,21 @@
    [react/view styles/edit-profile-name-container
     [profile-name-input]]])
 
-(defn edit-profile-status [{:keys [status edit-status?]}]
-  (let [input-ref (reagent/atom nil)]
+(defview edit-profile-status []
+  (letsubs [edit-status? [:my-profile/get :edit-status?]
+            status       [:my-profile/get :status]]
     [react/view styles/edit-profile-status
      [react/scroll-view
       (if edit-status?
         [react/text-input
-         {:ref               #(reset! input-ref %)
-          :auto-focus        edit-status?
+         {:auto-focus        (if edit-status? true false)
```

Comment:
isn't this just `edit-status?` would consider renaming it to  `auto-focus?`

Reply:
I don't know about this one, the focus can be on something else, auto-focus in the app-db wouldn't be really helpful to figure our what it implies. Here it means user is editing the status field.